content/livesync/outbox-node-tables.textile

```diff
@@ -1,39 +1,139 @@
 ---
-title: Database setup
-meta_description: "The Ably Database Connector requires two tables in your database to run. See how to create each table and the privileges required to utilize them."
+title: Outbox and Node tables
+meta_description: "LiveSync requires two tables, the outbox table and a nodes table to reflect database changes on the client's frontend."
 product: livesync
 ---
 
-The Ably Database Connector requires two tables in your database: an "outbox table":#outbox and a "nodes table":#nodes.
+LiveSync requires two database tables in your database, an "outbox table":#outbox-table and a "nodes table":#nodes-table.
 
-h2(#create). Create tables
+h2(#outbox-table). Outbox table
 
-When starting the Database Connector, you can manually create the required tables using the "CLI":#cli or "automatically":#auto.
+The Database Connector utilizes the "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html to send realtime, application-defined change events via Ably's global edge messaging platform. The outbox pattern allows for the reliable publication of an ordered sequence of change event messages over Ably, providing exactly-once delivery guarantees. This pattern involves the use of an outbox table that you create in your database.
+
+The Database Connector automatically monitors new records written to the outbox table and publishes them as messages over "channels":/channels.
+
+When your backend server processes a request to update data in your database, it should also write a corresponding change event to the outbox table within the same transaction as the underlying mutation. The Database Connector then publishes the change event, written to the outbox, to the channel in the exact order that the underlying change to the data is committed to the database. Furthermore, the Database Connector automatically deletes records from the table once the records are successfully processed.
+
+This example adds a comment and logs the event in an outbox:
+
+```[text]
+BEGIN;
+-- mutate your data, e.g.:
+INSERT INTO comments (comment) VALUES ('New comment!');
+-- write change event to outbox, e.g.:
+INSERT INTO outbox (mutation_id, channel, name, data) VALUES ('my-mutation-id', 'posts:123', 'addComment', 'New comment!');
+COMMIT;
+```
+
+The following describes the columns in the outbox table:
+
+- @mutation_id@ := is the identifier for the mutation.
+- @channel@ := is the channel to deliver the mutation, in this case, @posts:123@.
+- @name@ := indicates a comment is added.
+- @data@ := is the actual content or details of the mutation.
```

Comment:
Data _has_ to be a json encodable object. (not a single JSON value or a JSON array)